### ISSUE TITLE
Training on my dataset

### ISSUE THREAD

**Issue by JustAnotherPersonD** [karolzak/keras-unet#1]
When I train on my own dataset, the loss and dice_coef increases and iou also increases. I tried a range of learning rates, but it didn't help. Please help me out. In what format are the images and labels?
